cmd/sriovdp/manager.go

```diff
@@ -131,11 +208,21 @@ func (rm *resourceManager) initServers() error {
 			glog.Infof("no devices in device pool, skipping creating resource server for %s", rc.ResourceName)
 			continue
 		}
+
 		rPool, err := rm.rFactory.GetResourcePool(rc, filteredDevices)
 		if err != nil {
 			glog.Errorf("initServers(): error creating ResourcePool with config %+v: %q", rc, err)
 			return err
 		}
+
+		if rm.useCdi {
+			err = cdi.CreateCDISpec(rm.resourcePrefix, filteredDevices, rPool)
```

Comment:
I see you regenerated it because the logic is in device plugin and you don't want to duplicate it